src/gpuhunt/providers/datacrunch.py

```diff
@@ -61,40 +61,41 @@ def _transform_instance(instance: InstanceType, spot: bool, location: str) -> di
     return raw
 
 
+GPU_MAP = gpu_map = {
```

Comment:
I guess a second `gpu_hunt` is unnecessary

Reply:
Thanks!